cases/tests/tests.py

```diff
@@ -364,3 +373,162 @@ def test_compile_dates_correctly_uses_the_current_timezone():
     start, end = compile_dates(data)
     assert start == expected_start
     assert end == expected_end
+
+
+def test_log_files_happy_path(admin_client, case_1, action_types):
```

Comment:
Wonder if a separate cases/tests/test_log_files.py might be nicer? Doesn't really matter.